src/DIPS.Xamarin.UI/Controls/TrendGraph/TrendAnimation.cs

```diff
@@ -0,0 +1,29 @@
+namespace DIPS.Xamarin.UI.Controls.TrendGraph
+{
+    /// <summary>
+    ///     Animation types for <see cref="TrendGraph" />
+    ///     <remarks>Default is <see cref="None" /></remarks>
+    /// </summary>
+    public enum TrendAnimation
+    {
+        /// <summary>
+        ///     No animation
+        /// </summary>
+        None = 0,
+
+        /// <summary>
+        ///     Growing bars with faster growing background animation
+        /// </summary>
+        GrowAll,
+
+        /// <summary>
+        ///     Growing bars with faster fading background animation
+        /// </summary>
+        GrowAndFade,
+
+        /// <summary>
+        ///     Growing bars with fixed background animation
```

Comment:
"With **no** background animation" (Sounds better as you have called it GrowAnd**No**ne)

Reply:
used "with **not changing** background animation"